client/js/controllers/eventController.js

```diff
@@ -0,0 +1,26 @@
+apptivistApp.controller('eventController',['$http', 'EventFactory', 'EventService', function ($http, EventFactory, EventService) {
+  var self = this;
+
+  self.events = [];
+
+  self.updateEvents = function() {
+    EventService.getEvents().then(function(events){
+      self.events = events;
+    });
+  };
+
+  self.updateEvents();
+
+  self.sendEvent = function(title, description, time, location) {
+    self.postEvent(self.createEvent(title, description, time, location));
+  };
+
+  self.postEvent = function(eventData){
+    $http.post('/api/events', eventData, { 'Content-Type': 'application/json;charset=UTF-8' });
```

Comment:
add a call to self.updateEvents() as a promise to have the page content update once a new event has been sent